mappings/net/minecraft/client/realms/RealmsProxyConfig.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4344 net/minecraft/client/realms/RealmsClientConfig
+CLASS net/minecraft/class_4344 net/minecraft/client/realms/RealmsProxyConfig
```

Comment:
The `RealmsClientConfig` was based on the old name when the Realms jar was unobfuscated, so it seems that Mojang is using this class for storing all realms client config, but it just happens that the only config option right now is the proxy.

Reply:
Having a separate class specifically for proxy config is weird, while having a config class makes more sense (and it's what it actually is). Matching the actual name will also make sure that it will stay accurate as Mojang possibly adds more things to it.